src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta 
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    >
+    <link rel="preconnect" 
+          href="https://fonts.gstatic.com"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" 
+      rel="stylesheet"
+    >
+    <title>Product cards</title>
+    <link 
+      rel="stylesheet" 
+      href="./style.css"
+    >
+  </head>
+  <body>
+    <div 
+      class="card"
+      data-qa="card"
+    >
+      <img 
+        src="./images/imac.jpeg" 
+        alt="images"
+        class="card__image"
+      >
+      <p class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
```

Comment:
looks like heading

Reply:
changen it / tag <p> was changed to tag <h1>